packages/fxa-auth-server/config/index.ts

```diff
@@ -387,6 +389,40 @@ const conf = convict({
       default: '',
       env: 'SES_CONFIGURATION_SET',
     },
+    bounces: {
+      enabled: {
+        doc: 'Flag to enable checking for bounces before sending email',
+        default: true,
+        env: 'BOUNCES_ENABLED',
+      },
+      complaint: {
+        doc: 'Tiers of max allowed complaints per amount of milliseconds',
+        default: {
+          // 0 are allowed in the past day.
+          // 1 is allowed in the past week.
+          0: ONE_DAY,
+          1: 7 * ONE_DAY,
+        },
+        env: 'BOUNCES_COMPLAINT',
+      },
+      hard: {
+        doc: 'Tiers of max allowed hard bounces per amount of milliseconds',
+        default: {
+          // 0 are allowed in the past day.
+          // 1 is allowed in the past two days.
+          0: ONE_DAY,
+          1: 2 * ONE_DAY,
```

Comment:
used to be 1 year. no wonder why people have problems with bounces 😳

Reply:
Yeah I [recommended changing these values](https://docs.google.com/document/d/1lXa6ZwxtyXbzB-YQOr0HuRRwtKzqE2etiWUwEN9IRok/edit?disco=AAAAPRMDFnc) as a short-term fix when I was poking around for #10527 .

Reply:
I didn't have a specific suggestion, as I don't understand SES' deliverability score too well and how lowering the threshold (especially substantially) might affect it. If you do lower the threshold, I would let Janet/Anna/Chelsea know, as it may substantively impact (hopefully improve?) our bounce rates or other email delivery metrics.